zbus/src/connection/builder.rs

```diff
@@ -88,6 +89,22 @@ impl<'a> Builder<'a> {
         Self::address(&crate::address::system()?)
     }
 
+    /// Create a builder for an IBus connection
+    pub fn ibus() -> Result<Self> {
+        let ibus_address = {
+            let output = Command::new("ibus")
+                .arg("address")
+                .output()
+                .expect("Fail to run `ibus address`");
```

Comment:
This needs to be async. There is a good reason why I asked you to base this on https://github.com/dbus2/zbus/pull/976 . You at least need the first 2 commits from there. Please rebase this branch on that PR's branch and make use of the `Command` API introduced there.

Reply:
I see you didn't merge it to your main yet.

Which branch should I rebase on?

Reply:
> I see you didn't merge it to your main yet.

Yeah, there is still some minor stuff needed there but that's only about the later commits, which shouldn't affect your work.

> Which branch should I rebase on?

The [repo+branch of the PR](https://github.com/vially/zbus/tree/unixexec).